#!/usr/bin/python3
print ("pyhton exceptions")